extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -65,11 +66,15 @@ private YoutubeParsingHelper() {
     }
 
     public static final String YOUTUBEI_V1_URL = "https://www.youtube.com/youtubei/v1/";
+    public static final String CPN = "cpn";
```

Comment:
As you know what CPN stands for, please add that info as JDoc.
You might also want to link the alphabet and corresponding method and vice versa.